### PR TITLE
Feature/date filter on portal

### PR DESCRIPTION
**Pull request by chloe-yuu**

### CODE REVIEW

File: packages/bcer-data-portal/app/src/views/Locations.tsx [hunk 1]
```diff
@@ -513,11 +561,13 @@ export default function Locations() {
                   noi_report: searchTerms.noi_report ? searchTerms.noi_report: 'all',
                   product_report: searchTerms.product_report ? searchTerms.product_report: 'all',
                   manufacturing_report: searchTerms.manufacturing_report ? searchTerms.manufacturing_report: 'all',
-                  sales_report: searchTerms.sales_report ? searchTerms.sales_report: 'all'
+                  sales_report:  searchTerms.sales_report ? searchTerms.sales_report: 'all',
+                  fromdate: searchTerms.fromdate ? searchTerms.fromdate : 'all',
```
Comment (alateefah) [inline packages/bcer-data-portal/app/src/views/Locations.tsx:565]: You shouldn't use 'all' here, confirm from mui docs what empty value is for calender. Should be "" or null

File: packages/bcer-data-portal/app/src/views/Locations.tsx [hunk 2]
```diff
@@ -513,11 +561,13 @@ export default function Locations() {
                   noi_report: searchTerms.noi_report ? searchTerms.noi_report: 'all',
                   product_report: searchTerms.product_report ? searchTerms.product_report: 'all',
                   manufacturing_report: searchTerms.manufacturing_report ? searchTerms.manufacturing_report: 'all',
-                  sales_report: searchTerms.sales_report ? searchTerms.sales_report: 'all'
+                  sales_report:  searchTerms.sales_report ? searchTerms.sales_report: 'all',
+                  fromdate: searchTerms.fromdate ? searchTerms.fromdate : 'all',
+                  todate: searchTerms.todate ? searchTerms.todate : 'all'
```
Comment (alateefah) [inline packages/bcer-data-portal/app/src/views/Locations.tsx:566]: Same as from date

File: packages/bcer-data-portal/app/src/views/Locations.tsx [hunk 3]
```diff
@@ -582,6 +632,42 @@ export default function Locations() {
...
+                    </Grid>   
+                    <Grid item md={2} xs={6}>
+                      To
+                      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                        <KeyboardDatePicker
+                          className={classes.root}
+                          inputProps={{ className: classes.picker }}
+                          TextFieldComponent={TextFieldComponent}
+                          format="MM/dd/yyyy"
+                          value={selectedToDate ? moment(selectedToDate) : null}
```
Comment (alateefah) [inline packages/bcer-data-portal/app/src/views/Locations.tsx:661]: You are using null here, so the previous review should probably be null not "" but please confirm from documentation and adjust accordingly

**Review by alateefah — CHANGES_REQUESTED** [review]

File: packages/bcer-data-portal/app/src/views/Locations.tsx (hunk 1 above)
Comment (chloe-yuu) [reply packages/bcer-data-portal/app/src/views/Locations.tsx:565]: should be null, updated

**Pull request merged**